cmd/ocm/list/rhRegion/cmd.go

```diff
@@ -0,0 +1,40 @@
+package rhRegion
+
+import (
+	"fmt"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+
+	"github.com/openshift-online/ocm-cli/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "rh-regions",
+	Short:   "List available OCM regions",
+	Long:    "List available OCM regions",
+	Example: `  # List all supported OCM regions ocm list rh-regions`,
```

Comment:
should this be `ocm list rh-regions` only?

Reply:
Example sections in other files have the same format : https://github.com/openshift-online/ocm-cli/blob/main/cmd/ocm/list/version/cmd.go#L38

Reply:
Okay, but IMO that's not really helpful.